code/modules/reagents/reactions/reaction_compounds.dm

```diff
@@ -146,3 +146,15 @@
 	name = "Sulfuric Acid"
 	required_reagents = list(/decl/material/solid/sulfur = 1, /decl/material/solid/sodium = 2)
 	result_amount = 3
+
+/decl/chemical_reaction/compound/condensed_capsaicin
```

Comment:
is there a reason this doesn't use heating_products anymore?

Reply:
Can you not just increase the heating product temperature?

Reply:
But fire sources have a draught. I guess maybe you wouldn't be able to cook soup on a campfire then? The other thing is like, technically a pot full of water shouldn't be able to heat above the boiling temperature of water, but that's kind of a pain to check.
Also there's no inhibitors set for this so I don't know how this is any different than heating products, do growns have NOREACT or something? I guess they must to avoid being able to inject water into a banana for a quick explosion

Reply:
Oh, I missed the max temperature bit. Huh. Fair enough